customize-direct-manipulation.php

```diff
@@ -168,6 +177,14 @@ public function maybe_add_page_menu_class( $args ) {
 		return $args;
 	}
 
+	public function make_footer_credit_discoverable( $link ) {
+		$credit_link_regex = '#(<a)([^>]*href="https?://(?:www\.)?wordpress\.(?:com|org)[^"]*"[^>]*>)#i';
+
+		$link = preg_replace( $credit_link_regex, '$1 data-type="cdm-footer-credit-link"$2', $link );
```

Comment:
Since we control the footer-credit plugin, why not just skip all of this and add something discoverable and unique to the tag that the plugin creates? That way it could potentially be useful other places as well. I'll ping you in Slack about where the plugin lives.